src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -97,7 +95,7 @@ export default function FormNav(props) {
 
   return (
     <div>
-      <SegmentedProgressBar total={chapters.length} current={current} />
+      <va-segmented-progress-bar total={chapters.length} current={current} />
```

Comment:
Can we get a browser screenshot of what a form page looks like with this in the DOM?

Reply:
I updated it in the Screenshots section of the main thread

Reply:
This document page will need to be updated as well - https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/forms/available-features-and-usage-guidelines/#progress-bar

Reply:
Thanks @Mottie I have created the corresponding PR here: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/713